src/actions/members.js

```diff
@@ -104,6 +105,101 @@ async function getStatsDone(handle, uuid, tokenV3) {
   return { data, handle, uuid };
 }
 
+/**
+ * Payload creator for the action that inits the loading of member active challenges.
+ * @param {String} handle
+ * @param {String} uuid
+ * @returns {Object} Payload
+ */
+async function getActiveChallengesInit(handle, uuid) {
+  return { handle, uuid };
+}
+
+/**
+ * Payload creator for the action that loads the member active challenges.
+ * @param {String} handle
+ * @param {String} uuid
+ * @param {String} tokenV3
+ * @returns {Object} Payload
+ */
+async function getActiveChallengesDone(handle, uuid, tokenV3) {
+  const filter = { status: 'ACTIVE' };
+  const service = getChallengesService(tokenV3);
+  /* TODO: Reuse `getAll` from `actions/challenge-listing`
```

Comment:
Do I get it right, looking at the reducer, that it is actually only used to get the count of active challenges (see the reducer)? In such case we must:
1. Rename related actions / payload creators / etc. into `countActiveChallenges...`
2. Don't fetch all active challenges to count them! It is enough to hit challenge API with 0 limit: in all cases the response will include the total count of matched challenges, even if none are returned: https://github.com/topcoder-platform/topcoder-react-lib/blob/master/src/services/challenges.js#L383
3. The actual code for such counting should be placed into `services/challenges` module